theme.json

```diff
@@ -0,0 +1,413 @@
+          "font-size": "clamp(1.57rem, calc(1.04rem + 2.69vw), 3.81rem)",
+          "letter-spacing": "-0.01em"
+        }
+      },
+      "space": {
+        "padding": {
+          "global": "clamp(1.00rem, calc(0.40rem + 3.01vw), 3.50rem)"
+        },
+        "block": {
+          "gap": "clamp(1.50rem, calc(1.38rem + 0.60vw), 2.00rem)"
+        }
+      },
+      "alignment": {
+        "center": "744px",
+        "wide": "1008px"
+      },
+      "background": {
+        "primary": "#f5f2f1"
+      },
+      "text": {
+        "primary": "#13191e",
+        "secondary": "#444140",
+        "helper": "#6a6766",
+        "interactive": "#9a6014"
+      },
+      "link": {
+        "default": "#9a6014",
+        "hover": "#7a4909"
+      },
+      "button": {
+        "primary": {
+          "default": "#13191e",
+          "hover": "#d8a45f",
+          "active": "#ba7920",
+          "label": "#fcfafa"
+        }
+      }
+    },
+    "layout": {
+      "contentSize": "var(--wp--custom--alignment--center)",
+      "wideSize": "var(--wp--custom--alignment--wide)"
+    },
+    "spacing": {
+      "margin": false,
+      "padding": false,
+      "units": ["rem", "%"]
+    },
+    "typography": {
+      "customFontSize": false,
+      "fontStyle": false,
+      "fontWeight": false,
+      "letterSpacing": false,
+      "lineHeight": false,
+      "textDecoration": false,
+      "textTransform": false,
+      "dropCap": false,
+      "fontFamilies": [],
+      "fontSizes": []
+    },
+    "blocks": {
+      "core/heading": {
+        "typography": {
+          "fontSizes": [
+            {
+              "slug": "heading-1",
+              "size": "var(--wp--custom--heading--1--font-size)",
+              "name": "Heading 1"
+            },
+            {
+              "slug": "heading-2",
+              "size": "var(--wp--custom--heading--2--font-size)",
+              "name": "Heading 2"
+            },
+            {
+              "slug": "heading-3",
+              "size": "var(--wp--custom--heading--3--font-size)",
+              "name": "Heading 3"
+            },
+            {
+              "slug": "heading-4",
+              "size": "var(--wp--custom--heading--4--font-size)",
+              "name": "Heading 4"
+            },
+            {
+              "slug": "heading-5",
+              "size": "var(--wp--custom--heading--5--font-size)",
+              "name": "Heading 5"
+            }
+          ]
+        },
+        "color": {
+          "gradients": [],
+          "text": true,
+          "palette": [
+            {
+              "slug": "black",
+              "color": "var(--wp--custom--text--primary)",
+              "name": "Black"
+            },
+            {
+              "slug": "gray",
+              "color": "var(--wp--custom--text--helper)",
+              "name": "Gray"
+            }
+          ]
+        }
+      }
+    }
+  },
+  "styles": {
```

Comment:
Lots of styles to add here, but it might be easer to create the basic system you've made here in Figma and do a quick design pass on all the core blocks to see what needs to get done. We'll need to do that eventually so that we can hook up the design bridge.

Reply:
I'm leaning towards that being better to do when we make the complete valet theme package vs when we are just making the `theme.json` as we are right now?

Lemme know if I'm missing something.